content/docs/guides/crosswalk/_index.md

```diff
@@ -1,4 +1,36 @@
 ---
-redirect_to: /docs/guides/crosswalk/aws/
+title: "Pulumi Crosswalk"
```

Comment:
IMHO, we don't need to add this extra level of nav -- it clutters things up a bit and deepens a hierarchy that's already too deep. In fact, I'm tempted to go in the _opposite_ direction, and eliminate the "Guides" parent nav, to help increase the visibility of this content ... worth discussing (and curious what @cnunciato thinks).

Reply:
I agree this page can probably be removed, since it's confusing to see it separate from and alongside *Crosswalk for AWS* and *Crosswalk for Kubernetes*. 

That said, an introductory Crosswalk overview page does seem useful -- here's what Crosswalk is, what it aims to do, what's currently available, etc. -- with links to the "for AWS" and "for Kubernetes" subsections underneath it.

Reply:
I've removed the index page as we all agree it seemed confusing.